app/views/posts/_feed.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="post-grid">
+
+<% @posts.each do | post | %>
+    <div class="box card mr-1 p-3 d-flex flex-column">
+            <div class="media-left d-flex">
+                <figure class="">
+                    <%= gravatar_image_tag(post.user.email,size: 15, alt: post.user.name, class: 'rounded-circle mr-2') %>
+                </figure>
+                <div class="content d-flex flex-column">
+                    
+                    <div class="d-flex flex-column m-0 p-0">
+                        <% if user_signed_in? %>
+                            <strong class="text-capitalize "> <%= post.user.name %> </strong>
+                            <small> @<%= post.user.username %> </small>
+                        <% else %>
+                            <strong> Anonymous User </strong>
+                            <small> @Anonymous </small>
+                        <% end %>
```

Comment:
There should be absolutely NO logic in views. Kindly use the helper file.
- [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
- [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
- [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright